### PR TITLE
[nbc] Fix downloading classic-tv videos

### PR DESCRIPTION
**Pull request by einstein95**
## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [adding new extractor tutorial](https://github.com/rg3/youtube-dl#adding-support-for-a-new-site) and [youtube-dl coding conventions](https://github.com/rg3/youtube-dl#youtube-dl-coding-conventions) sections
- [x] [Searched](https://github.com/rg3/youtube-dl/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into youtube-dl each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [x] Bug fix
- [ ] Improvement
- [ ] New extractor
- [ ] New feature

---

### Description of your *pull request* and other information

Adds `(?:classic-tv/|)` to the valid URL regex as suggested by a user in the IRC channel. An example is http://www.nbc.com/classic-tv/charles-in-charge/video/charles-in-charge-pilot/n3310. A US IP is required.

### CODE REVIEW

File: youtube_dl/extractor/nbc.py [hunk 1]
```diff
@@ -15,7 +15,7 @@
 
 
 class NBCIE(AdobePassIE):
-    _VALID_URL = r'https?(?P<permalink>://(?:www\.)?nbc\.com/[^/]+/video/[^/]+/(?P<id>n?\d+))'
+    _VALID_URL = r'https?(?P<permalink>://(?:www\.)?nbc\.com/(?:classic-tv/|)[^/]+/video/[^/]+/(?P<id>n?\d+))'
```
Comment (dstftw) [inline youtube_dl/extractor/nbc.py]: 1. If you want to make it optional you should use `?` qualifier.
2. Add a test.

**Comment by einstein95** [comment]
Done and done

File: youtube_dl/extractor/nbc.py [hunk 2]
```diff
@@ -68,6 +68,23 @@ class NBCIE(AdobePassIE):
             },
             'skip': 'Only works from US',
         }
```
Comment (dstftw) [inline youtube_dl/extractor/nbc.py]: Missing comma.

File: youtube_dl/extractor/nbc.py [hunk 3]
```diff
@@ -68,6 +68,23 @@ class NBCIE(AdobePassIE):
...
+        {
+            'url': 'https://www.nbc.com/classic-tv/charles-in-charge/video/charles-in-charge-pilot/n3310',
+            'info_dict': {
```
Comment (dstftw) [inline youtube_dl/extractor/nbc.py]: Make test only matching.

**Pull request merged**